src/packet/pacer.rs

```diff
@@ -1142,14 +1169,162 @@ mod test {
         Duration::from_millis(ms)
     }
 
-    fn make_packet(seq_no: u16, size: usize, kind: MediaKind) -> (RtpHeader, Vec<u8>, MediaKind) {
+    fn make_packet(seq_no: u16, size: usize, kind: MediaKind) -> (RtpHeader, usize, MediaKind) {
         let mut header = RtpHeader {
             sequence_number: seq_no,
             ..Default::default()
         };
-        let data = vec![0; size];
 
-        (header, data, kind)
+        (header, size, kind)
+    }
+
+    #[derive(Debug, Clone, Copy)]
+    struct RealisticTestConfig {
+        media_rate: Bitrate,
+        padding_rate: Bitrate,
+        duration: Duration,
+        // Spike probability as a percentage
+        spike_probability: u8,
+        max_overshoot_factor: f32,
+        frame_pacing: Duration,
+    }
+
+    impl Default for RealisticTestConfig {
+        fn default() -> Self {
+            RealisticTestConfig {
+                media_rate: Bitrate::kbps(250),
+                padding_rate: Bitrate::kbps(800),
```

Comment:
I am just thinking I had problems with 250kbps of media, to validate bandwidth up to 1500kbps. This could validate that scenario ? i.e. the ability to pad to that rate

Reply:
Yea, the actual test uses 2.5Mbit/s in fact.